casepro/cases/models.py

```diff
@@ -420,11 +465,21 @@ def access_level(self, user):
     def is_closed(self):
         return self.closed_on is not None
 
+    @property
+    def has_passed_response_time(self):
+        response_required_in = getattr(settings, 'SITE_CASE_RESPONSE_REQUIRED_TIME', None)
+        if response_required_in is None:
+            return False
+
+        minutes_since_reassigned = (now() - self.last_reassigned_on).total_seconds() // 60
+        return minutes_since_reassigned > response_required_in
+
     def as_json(self, full=True):
         if full:
             return {
                 'id': self.pk,
                 'assignee': self.assignee.as_json(full=False),
+                'user_assignee': self.user_assignee.as_json(full=False) if self.user_assignee else None,
```

Comment:
`user_assignee` should be prefetched in `Case.search(...)` (and anywhere else we fetch more than one case)